tests/test_polygons.py

```diff
@@ -246,3 +254,25 @@ def test_bounds(polygons, expected_bounds):
     assert close_enough(min_y, expected_min_y)
     assert close_enough(max_x, expected_max_x)
     assert close_enough(max_y, expected_max_y)
+
+
+@pytest.mark.parametrize('parents, children, expected_intersection_percentage', (
+    # Hackney Marshes is a small part of the Lea Valley
+    ([LEA_VALLEY], [HACKNEY_MARSHES], 1.865),
+    # … and the Lea Valley wholey contains Hackney Marshes
+    ([HACKNEY_MARSHES], [LEA_VALLEY], 100),
+    # Isle of Dogs is a wholey separate area from Hackney Marshes
+    ([ISLE_OF_DOGS], [HACKNEY_MARSHES], 0),
+    # A small part of the Isle of Dogs overlaps with the Lea Valley…
+    ([LEA_VALLEY], [ISLE_OF_DOGS], 1.187),
+    # …but as a proportion of the area of the Isle of Dogs, the overlap
+    # is larger
+    ([ISLE_OF_DOGS], [LEA_VALLEY], 7.115),
+    # Ratio is always 0 if one or both polygons are empty
+    ([ISLE_OF_DOGS], [], 0),
+    ([], [ISLE_OF_DOGS], 0),
+    ([], [], 0),
+))
+def test_intersection_ratio(parents, children, expected_intersection_percentage):
```

Comment:
Minor: these names don't accurately describe the data they represent, as some of the params swap between supersets and subsets, and overlaps.

We could just say `polygons_1` and `polygons_2` or similar.